src/components/containers/Disclosure/Disclosure.module.scss

```diff
@@ -0,0 +1,140 @@
+    
+    --bk-disclosure-transition-duration: 500ms;
+    
+    --bk-disclosure-background-color: #{bk.$theme-accordion-background-default};
+    --bk-disclosure-border-color: #{bk.$theme-accordion-border-default};
+    --bk-disclosure-text-color: var(--bk-page-color-text);
+    
+    --bk-disclosure-border-radius-top: #{bk.$radius-3};
+    --bk-disclosure-border-radius-bottom: #{bk.$radius-3};
+    --bk-disclosure-padding-inline: #{bk.$spacing-4};
+    --bk-disclosure-padding-block: #{bk.rem-from-px(14)}; // FIXME: does not correspond to an existing token
+    
+    overflow: hidden;
+    
+    max-height: var(--bk-disclosure-max-height, none);
+    padding: var(--bk-disclosure-padding-block) var(--bk-disclosure-padding-inline);
+    
+    border: bk.$size-1 solid var(--bk-disclosure-border-color);
+    border-radius:
+      var(--bk-disclosure-border-radius-top)
+      var(--bk-disclosure-border-radius-top)
+      var(--bk-disclosure-border-radius-bottom)
+      var(--bk-disclosure-border-radius-bottom);
+    background: var(--bk-disclosure-background-color);
+    color: var(--bk-disclosure-text-color);
+    
+    // Use flex so that we can fit the content to the <details> height, allowing us to do `overflow-y: auto`
+    display: flex;
+    flex-direction: column;
+    
+    summary {
+      @include heading.h5;
+      
+      cursor: pointer;
+      
+      // Expand the target size
+      margin: calc(-1 * var(--bk-disclosure-padding-block)) calc(-1 * var(--bk-disclosure-padding-inline));
+      padding: var(--bk-disclosure-padding-block) var(--bk-disclosure-padding-inline);
+      
+      display: flex;
+      align-items: center;
+      gap: bk.$spacing-9;
+      
+      line-height: bk.$line-height-s;
+      
+      .bk-disclosure__title {
+        @include bk.text-one-line;
+      }
+      
+      .bk-disclosure__collapse-icon {
+        margin-inline-start: auto;
+        font-size: 1.125em; // ~18px
+        
+        @media (prefers-reduced-motion: no-preference) {
+          transition: rotate 120ms ease-in-out;
+        }
+      }
+      
+      // Border radius for focus outline
+      border-radius: // Note: `inherit` keyword does not work for some reason?
+        var(--bk-disclosure-border-radius-top)
+        var(--bk-disclosure-border-radius-top)
+        var(--bk-disclosure-border-radius-bottom)
+        var(--bk-disclosure-border-radius-bottom);
+      @include bk.focus-inset;
+    }
+    &[open] summary {
+      --bk-disclosure-border-radius-bottom: 0; // Disable bottom radius on the summary when open (for focus outline)
+      
+      .bk-disclosure__collapse-icon {
+        rotate: -0.5turn;
+      }
+    }
+    
+    // Inner wrapper for content. We cannot give `::details-content` any margin/padding because it would still be
+    // visible (due to `content-visibility: hidden` still taking up space). So we use this as scroller and for spacing.
+    .bk-disclosure__content {
+      overflow-y: scroll;
```

Comment:
Scroll bar is displayed inside of all accordion items, even though it's not scrollable. 
Is it better to hide this scroll bar?
![Screenshot from 2024-12-23 10-48-47](https://github.com/user-attachments/assets/9467a458-810d-44d3-95eb-d54ae6e889d9)

Reply:
Ah yeah, good catch. I was testing this on Mac where the scrollbars aren't shown anyway. Needed to use `overflow-y: auto` instead. Fixed now.

Reply:
With `overflow-y: auto` change, scroll bar is hidden when accordion item is fully open. 
But while we open or close it, the scroll bar is displayed (please refer the video below.).

https://github.com/user-attachments/assets/6acd2708-845c-455b-bab8-a27c0874ffc8

Reply:
@spli02 You're right that's a bit jarring. I pushed a fix, now the `overflow-y: auto` only gets applied after the transition is complete on open, and it gets removed immediately upon triggering close.